modules/@apostrophecms/page/index.js

```diff
@@ -1977,14 +1977,13 @@ database.`);
         );
       },
       // Returns the effective base URL for the given request.
-      // If Apostrophe's top-level `baseUrl` option is set, it is returned,
-      // otherwise the empty string. This makes it easier to build absolute
+      // If Apostrophe's top-level `baseUrl` option is set, or a hostname for the
```

Comment:
"or a hostname for the active locale" seems missing a verb. "or a hostname is defined for the active locale"?